src/screens/AdminPortal/EventVolunteers/Volunteers/createModal/VolunteerCreateModal.tsx

```
@@ -31,7 +31,7 @@ import React, { useState, useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useMutation, useQuery } from '@apollo/client';
 import { NotificationToast } from 'components/NotificationToast/NotificationToast';
-import { Autocomplete } from '@mui/material';
+import Autocomplete from '@mui/material/Autocomplete';
```

ESLint

[error] '@mui/material/Autocomplete' import is restricted from being used. Do not import Autocomplete from `@mui/material`. Use the shared Autocomplete component from src/shared-components/Autocomplete/ instead.

(no-restricted-imports)